aws/routes.go

```diff
@@ -34,6 +34,7 @@ func init() {
 				Summary:     "get aws accounts' data",
 				Description: "Gets the data for all of the user's AWS accounts.",
 			},
+			routes.RequiredQueryArgs{AwsAccountsOptionalQueryArg},
```

Comment:
The decorator was named `RequiredQueryArgs` because it was expected there would be an `OptionalQueryArgs` to go along with it. Since this PR introduces the `Optional` field, this doesn’t make sense anymore. Please rename this decorator to just `QueryArgs`.